frontend/src/components/ListView.vue

```diff
@@ -1,31 +1,29 @@
 <template>
 	<ion-header class="ion-no-border">
 		<div class="w-full sm:w-96">
-			<div
-				class="flex flex-row bg-white shadow-sm py-4 px-3 items-center justify-between border-b"
-			>
+			<div class="flex flex-row bg-white shadow-sm py-4 px-3 items-center justify-between border-b">
 				<div class="flex flex-row items-center">
-					<Button
-						variant="ghost"
-						class="!pl-0 hover:bg-white"
-						@click="router.back()"
-					>
+					<Button variant="ghost" class="!pl-0 hover:bg-white" @click="router.back()">
 						<FeatherIcon name="chevron-left" class="h-5 w-5" />
 					</Button>
 					<h2 class="text-xl font-semibold text-gray-900">{{ pageTitle }}</h2>
 				</div>
 
 				<div class="flex flex-row gap-2">
-					<Button
-						id="show-filter-modal"
-						icon="filter"
-						variant="subtle"
-						:class="[
-							areFiltersApplied
-								? '!border !border-gray-800 !bg-white !text-gray-900 !font-semibold'
-								: '',
-						]"
-					/>
+					<Button id="show-filter-modal" icon="filter" variant="subtle" :class="[
+						areFiltersApplied
+							? '!border !border-gray-800 !bg-white !text-gray-900 !font-semibold'
+							: '',
+					]" />
+					<Dropdown :options="sortOptions">
```

Comment:
This still doesn't have separate options for selecting the sort order and sort field in the dropdown.
![image](https://github.com/frappe/hrms/assets/24353136/8ae3f0e3-f5dd-427c-8ddd-d646bfeb6d7d)